engine/src/main/resources/swagger/cromwell.yaml

```diff
@@ -229,6 +239,29 @@ paths:
       security:
         - google_oauth:
             - openid
+  '/api/workflows/{version}/{id}/release':
```

Comment:
On reflection I'm not 100% sure on `release` - it almost sounds like we're asking Cromwell to release the workflow (rather than release the hold)

FWIW from thesaurus.com...:
<img width="763" alt="screen shot 2018-04-13 at 11 29 04 am" src="https://user-images.githubusercontent.com/13006282/38743792-f94621ae-3f0d-11e8-9d52-bdeacda687ec.png">

Reply:
`/api/workflows/{version}/{id}/let%20go` FTW :P

Reply:
I think `releaseHold` would also work

Reply:
I'd probably take `release` over any of these suggestions except possibly `releaseHold`

Reply:
I will rename it to 'releaseHold'